gitpod-io/gitpod
# [JetBrains] Replace "Status" by "JetBrains Launcher" and run it in a separate image from IDEs

## Description
This PR extracts the `status` program from the JB IDE images. We're adding a new image called `jb-launcher` and it will make iterating on this logic much faster, avoiding to re-build the images each time.

## Related Issue(s)

Related to #6740 

## How to test

1. Open the Preview Environment from this PR: https://ide-jetbra3bd70cdaa4.preview.gitpod-dev.com/workspaces
2. For Stable and Latest IntelliJ IDEA, confirm if:
    1. Prebuilds are working as usual using this branch: https://github.com/andreafalzetti/gitpod-prebuilds/tree/jb-3 (You can [join this Team](https://ide-jetbra3bd70cdaa4.preview.gitpod-dev.com/teams/join?inviteId=7b597208-d1c7-4d4d-9271-e42a14830e59) in the Preview Environment)
    2. The IDE connects to the workspace and has Gitpod Plugin installed.

## Release Notes

```release-note
NONE
```

## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
